src/config/Config.ts

```diff
@@ -1,3 +1,10 @@
+declare global {
```

Comment:
Instead of adding this permanently you should add it only for default 'dev' type deployment. 
We should add new env variable:
DEPLOY='dev'

When deploy == dev the possibility to set BACKEND_URL via console should be active

When deploy == prod then it should be disactivated.

Reply:
done